### PR TITLE
Pupa Conversion of NV

### PR DESCRIPTION
**Pull request by hiteshgarg14**
#1442 

Completed Pupa Conversion of #1600

### CODE REVIEW

**Comment by hiteshgarg14** [comment]
I will comment review of scraped data by tomorrow.

File: openstates/nv/__init__.py [hunk 1]
```diff
@@ -1,178 +1,155 @@
...
 
+    def get_session_list(self):
+        import re
+        return [re.sub(r'(\xa0|\(click to close\)|\(click to open\))', '', x.text_content())
+                for x in url_xpath('http://www.leg.state.nv.us/Session/',
+                                   '//*[@class="MainHeading"]')]
 
-def extract_text(doc, data):
-    return text_after_line_numbers(pdfdata_to_text(data))
+    def get_extract_text(self, doc, data):
```
Comment (jmcarp) [inline openstates/nv/__init__.py]: Is this used?

Comment (hiteshgarg14) [inline openstates/nv/__init__.py]: Nope. Just wanted to keep it. Should I remove it?

**Comment by hiteshgarg14** [comment]
Votes:-
votes some files only found in new
    ('SB 201', 'Assembly Final Passage')
    ('AB 477', 'Assembly Final Passage')
    ('AB 458', 'Assembly Final Passage')
    ('SJR 10', 'Assembly Final Passage')
    ('AB 309', 'Assembly Final Passage')
    ('AB 481', 'Assembly Final Passage')
    ('AB 449', 'Assembly Final Passage')
    ('AB 29', 'Assembly Final Passage')
    ('AB 322', 'Assembly Final Passage')

Bills:-
-  ```
      bills some files only found in old
           79_upper_SB261.json
           79_upper_AB480.json
           79_lower_SB33.jso
       bills some files only found in new
           79_lower_AB449.json
           79_upper_AB83.json
           79_upper_SB472.json
           79_upper_SB180.json
           79_upper_SB377.json
   ```
- Difference in `action` dates. (might be problem with timezone, please verify that one)

 
Rest Data Is Good To Go :+1:

**Pull request merged**